### DISCUSSION TITLE
다크모드 코드하이라이팅 텍스트 컬러 변경안됨

### DISCUSSION THREAD

**Discussion by heli-os** [pronist/hELLO#77] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: macOS 11.4 (20F71)
- 브라우저: Chrome Version 95.0.4638.69 (Official Build) (arm64)
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 3.4.5

**설명**
다크모드에서 코드 하이라이팅 시 텍스트 색깔이 어둡습니다.
배경과 유사한 색상이라 텍스트가 보이지 않습니다.

**발생 과정**
다크 모드로 코드 하이라이팅 확인

**스크린샷 및 링크**
<img width="514" alt="Screen Shot 2021-11-09 at 11 27 10 PM" src="https://user-images.githubusercontent.com/44586666/140942085-dacd8894-8e1d-461d-9012-8b3239505112.png">

저는 임시 방편으로 아래 방법으로 해결하여 사용중입니다.

```
html[data-theme='dark'] .content__permalink>.content .contents_style>pre>code
에 아래 style  추가    
color: #fafafa !important;
```

백엔드 개발자라 확실하진 않지만 아마 아래 코드가 아닐까 추정됩니다. 
https://github.com/pronist/hELLO/blob/9aa9ddbbb0518617ee086f635c6c81a16946c64c/assets/styl/Theme/Dark/App/Main/Post/_Post/_Content.styl#L60-L61

**Comment by pronist** [comment, marked as answer]
@960813 안녕하세요? 해당 증상이 발생하는 글 주소를 알 수 있을까요? 제 블로그에서는 일단은 정상으로 나오고 있고, 관련된 이슈는 예전에 처리된 이후 처음인지라 증상이 발생하는 글을 직접 보아야 할 것 같습니다.

추가적으로 #30 에 해당하는 이슈도 참고부탁드립니다!

**Comment by heli-os** [comment]
@pronist 
안녕하세요. 링크 걸어주신 이슈 기반으로 확인해보니 아래 플러그인의 문제인 것 같습니다. 티스토리 기본 Syntax Highlight 입니다.
https://notice.tistory.com/2483

해당 플러그인 사용을 중지하니 정상 표시되네요.
